gooddata-sdk/docs/services/catalog-workspace-content.rst

```diff
@@ -86,22 +86,22 @@ The *gooddata_sdk.catalog_workspace_content* supports the following declarative
                 └── workspaces
                         └── workspace_id
                                 └── analytics_model
-                                        ├── ldm
-                                        │    └── datasets
-                                        │           └── dataset.yaml
-                                        └── date_instaces
-                                                    └── date_instace.yaml
+                                        └── ldm
+                                            ├── datasets
+                                            │       └── dataset.yaml
+                                            └── date_instances
+                                                    └── date_instance.yaml
 
-* ``load_declarative_ldm(workspace_id: str, layout_root_path: Path = Path(os.path.curdir))``
+* ``load_declarative_ldm(workspace_id: str, layout_root_path: Path = Path.cwd())``
 
     Returns *CatalogDeclarativeModel*.
 
     Load declarative LDM layout, which was stored using *store_declarative_ldm*.
 
-* ``load_and_put_declarative_ldm(workspace_id: str, layout_root_path: Path = Path(os.path.curdir))``
+* ``load_and_put_declarative_ldm(workspace_id: str, layout_root_path: Path = Path.cwd(), validator: Optional[DataSourceValidator])``
 
     This method combines *load_declarative_ldm* and *put_declarative_ldm*
-    methods to load and set layouts stored using *store_declarative_ldm*.
+    methods to load and set layouts stored using *store_declarative_ldm*. You can pass an additional validator parameter, which checks that for every data source id in the logical data model the corresponding data source exists.
```

Comment:
You can pass an additional validator parameter which checks that for every data source id in the logical data model the corresponding data source exists.